hack/build-test-images.sh

```diff
@@ -6,14 +6,19 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+readonly S2I_CONTAINER_MANAGER="${S2I_CONTAINER_MANAGER:-docker}"
```

Comment:
I won't make the comment everywhere, but again on my container manager vs. oci tool point